sdk/evaluation/azure-ai-evaluation/setup.py

```diff
@@ -86,5 +86,6 @@
     },
     package_data={
         "pytyped": ["py.typed"],
+        "azure.ai.evaluation.simulator._prompty": ["*.prompty"],
```

Comment:
I recommend adding this to `MANIFEST.in` as the evaluation prompty files are included there